CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+## `v10.5.0`
+
+### Critical Changes
+
+1. This release fixes a critical issue for LDAP users, installing a needed package on Debian 11
+   on build-time. Moreover, a race-condition was eliminated ([#2341](https://github.com/docker-mailserver/docker-mailserver/pull/2341)).
+2. A resource leak in `check-for-changes.sh` was fixed ([#2401](https://github.com/docker-mailserver/docker-mailserver/pull/2401))
+
+### Other Major Changes
+
+1. `SPAMASSASSIN_SPAM_TO_INBOX`'s default changed to `1`.
+2. Changedetector functionality was added to `SSL_TYPE=manual`-setups.
+3. Three new environment variables were introduced: `LOGWATCH_SENDER`, `ENABLE_DNSBL` and `ENABLE_SPAMASSASSIN_KAM`.
+4. There are plenty of bug fixes and documentation enhancements with this release.
```

Comment:
Should we add the corresponding PRs here too?

Reply:
solved in [21d6b2c](https://github.com/docker-mailserver/docker-mailserver/pull/2431/commits/21d6b2c8fe9f8dff77b666ce434a360a938f8bc6)